src/Flags.h

```diff
@@ -66,6 +66,9 @@ struct Flags {
   // under valgrind.
   std::string forced_uarch;
 
+  // Print version number and exit.
+  bool version;
```

Comment:
Rather than add this to `Flags`, which I'm trying to reduce/eliminate, let's just make it a static variable in `main.cc`.